services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminExecutionTask.java

```diff
@@ -627,8 +625,9 @@ private void handleAddVersion(AddVersion message) {
             replicationMetadataVersionId);
       }
     } else {
-      String regions = message.targetedRegions != null ? message.targetedRegions.toString() : null;
-      if (StringUtils.isNotEmpty(regions) && !RegionUtils.parseRegionsFilterList(regions).contains(this.regionName)) {
+      boolean skipConsumption = message.targetedRegions != null
+          && !message.targetedRegions.stream().map(Object::toString).collect(Collectors.toSet()).contains(regionName);
```

Comment:
Instead of `.collect().contains`, we can use `. anyMatch`

Reply:
Cool! Updated to `message.targetedRegions.stream().map(Object::toString).noneMatch(regionName::equals);`